src/harness/reference_models/geo/utils.py

```diff
@@ -71,10 +71,10 @@ def GridPolygon(poly, res_arcsec):
 
   res = res_arcsec / 3600.
   bounds = poly.bounds
-  lng_min = int(bounds[0] / res) * res
-  lat_min = int(bounds[1] / res) * res
-  lng_max = int(bounds[2] / res + 1) * res
-  lat_max = int(bounds[3] / res + 1) * res
+  lng_min = np.floor(bounds[0] / res) * res
+  lat_min = np.floor(bounds[1] / res) * res
+  lng_max = np.ceil(bounds[2] / res + 1) * res
```

Comment:
np.ceil(bounds[2] / res) * res

Reply:
I am sorry but I dont think  this is correct: 
Imagine that
  bounds[2] = 10 
  res = 1 degree (3600 arcsec)
Then you have:
  lng_max = 10 
Now when we build the array of points to check, we do not include lng_max in the test, although it corresponds to a point at the border.

If you want to save testing for points which are obviously outside, then my previous version with int() on the left and int( ... + 1) on the right is the most accurate. 
Now with the floor, we have one more set of points on the left that we include on the test although they are outside. And on the right the ceil() usually introduce one serie of more points as well.